projects/plugins/jetpack/extensions/blocks/seo/seo.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Block Editor - SEO feature.
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Seo;
+
+use Jetpack_Gutenberg;
+
+/**
+ * Register SEO plugin.
+ *
+ * @return void
+ */
+function register_plugins() {
```

Comment:
The SEO Tools module requires a working connection to WordPress.com:
https://github.com/Automattic/jetpack/blob/3a3c4adc1c0913bac224ddb832713bd6101fe1ab/projects/plugins/jetpack/modules/seo-tools.php#L8-L9

With that in mind, I think we should add a check like here:
https://github.com/Automattic/jetpack/blob/13262b644b6d7f90e6da48e752fcaa0e512327d5/projects/plugins/jetpack/extensions/plugins/publicize/publicize.php#L31-L36

This would avoid issues when you're trying to access the SEO Panel when in offline module, for example.

Reply:
Oh, I actually missed where to get this prerequisite data. I will know for the future. Thanks!